config/non-serializable/mailer.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   transport: {
-    host: 'localhost',
-    port: 25,
+    host: `${process.env.MAILER_HOST}`,
```

Comment:
both should have a default value?

Reply:
That might be kinder for the mis-configured cases. :+1: